config/jobs/kubernetes/sig-network/sig-network-kind.yaml

```diff
@@ -192,12 +192,10 @@ presubmits:
         env:
         - name: PARALLEL
           value: "true"
-        - name: FOCUS
-          value: \[sig-network\]|\[Conformance\]
         - name: KUBE_PROXY_MODE
           value: "nftables"
-        - name: SKIP
-          value: \[Feature:(Networking-IPv6|Example|Federation|PerformanceDNS|ServiceCIDRs)\]|LB.health.check|LoadBalancer|load.balancer|GCE|NetworkPolicy|DualStack
+        - name: LABEL_FILTER
+          value: "(Conformance || sig-network ) && !Feature: containsAny {Alpha, Beta, Networking-IPv6, Example, Federation, PerformanceDNS, LoadBalancer} && !Disruptive && !Flaky"
```

Comment:
based on this https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/127167/pull-kubernetes-e2e-kind-nftables/1832871323144032256
@adrianmoisey can you send a PR that  adds `IPv6DualStack` here, to the list of Features?